index.js

```diff
@@ -108,4 +106,7 @@ async function run() {
   console.log(`👍🏼  all releases deleted successfully!`);
 }
 
-run();
+if (shouldDeleteRelease) {
+  deleteReleases();
```

Comment:
```suggestion
  await deleteReleases();
```

@dev-drprasad do you just mean doing this?

Reply:
I don't think Node12 (used by GitHub actions) has top-level  `async`. we should wrap both `deleteReleases` and `deleteTag` functions in another `async` function or immediately invoked async function

ref: https://flaviocopes.com/javascript-await-top-level/

Reply:
I've set the PR to allow you to edit my change btw, so feel free to just commit that and merge if that's all it is :)

Reply:
Interesting. I could wrap the two back in a `main` as you had it originally, but that was not `await`ed either.

Reply:
`main` doesn't need to `await`ed. `main` should be `async`, so funcs inside it can be `await`ed

Reply:
I made a broader suggested change. Let me know if that's what you had in mind.